sql/core/src/test/scala/org/apache/spark/sql/CollationSQLExpressionsSuite.scala

```diff
@@ -2319,6 +2319,57 @@ class CollationSQLExpressionsSuite
     )
   }
 
+  // scalastyle:off nonascii
+  test("approx_count_distinct returns correct with collation") {
```

Comment:
```suggestion
  test("Support HyperLogLogPlusPlus expression with collation") {
```

let's follow the existing naming convention in this test suite